indico/modules/events/registration/util.py

```diff
@@ -104,6 +105,12 @@ def check_registration_email(regform, email, registration=None, management=False
     user = get_user_by_email(email)
     email_registration = regform.get_registration(email=email)
     user_registration = regform.get_registration(user=user) if user else None
+    signal = signals.event.before_check_registration_email
+    extra_checks = values_from_signal(signal.send(regform, email=email, registration=registration,
+                                                  management=management, user=user, user_registration=user_registration,
+                                                  email_registration=email_registration), as_list=True)
```

Comment:
```suggestion
    extra_checks = values_from_signal(
        signals.event.before_check_registration_email.send(
            regform,
            email=email, registration=registration, management=management, user=user,
            user_registration=user_registration, email_registration=email_registration
        ),
        as_list=True
    )
```

No strong preference though, so feel free to keep it as-is.